blogposts/2021/show-us-your-calendar.md

```diff
@@ -0,0 +1,166 @@
+
+TJ is frank: “I could work in the middle of the night for two nights a week and if I ship all the same amount of stuff, I don’t think people would care.” Though they might reach out about his health if it looked like he weren't sleeping, otherwise, his manager and his team just care about output.
+
+TJ recommends carrying over some software development principles over to your work calendar, namely: Don’t waterfall your calendar. “It’s unlikely that you’ll know the optimal schedule ahead of time,” TJ warns. “That’s not usually how we find the best version of something. We iterate on it over time.”
+
+## Robert Lin, software engineer, has a split schedule
+
+[Robert Lin](https://about.sourcegraph.com/handbook/company/team#robert-lin), software engineer at Sourcegraph, likes bouldering and hates queues.
+
+“For some activities,” he says, “you just can’t do them at night. And if you do it on weekends, then you’re competing with all these people who want to do the same thing on their two days off.”
+
+Robert works a split schedule, meaning that for three days a week, he works from 8 am to 12 pm and 6 pm to 10 pm. On the other two days, he works a normal 9 am to 5 pm. Robert, who’s currently in the United States, usually lives and works from Taiwan. There, he maintains a similar schedule, the hours shifted only slightly to maximize overlap with his American teammates.
+
+![Split schedule graphic](https://storage.googleapis.com/sourcegraph-assets/blog/Show%20Us%20Your%20Calendar%20Images/Split.png)
+
+Robert is used to some degree of flexibility. When he and his partner were students, they optimized their schedules for their primary hobbies–bouldering being number one. Robert decided he didn’t want to change that as he started working full time.
+
+When he onboarded with Sourcegraph, there was nothing that said he had to be working from 9 am to 5 pm so he set his schedule how he wanted it. “I don’t think there was any particular response to it,” he said.
+
+Robert is wary of the exhaustion that can come with a traditional schedule. “There’s something about 9 am to 5 pm that’s kind of exhausting,” he said. “What would often happen is you’d charge through the day and then you’d get home at the end of the workday, maybe hit the gym for a while, come back and cook dinner–and that’s pretty much your whole day. There’s no room for anything else.”
+
+Work is important to Robert but it isn’t his life–it’s just part of it: “The key thing for me is being able to structure work around things I want to do. That is super liberating.”
+
+Robert recommends building a company culture that prizes independence. A company culture where everyone needs everyone else to do anything is one that can’t accommodate different schedules. Robert says the first step to experimenting with flexibility is to “build up a culture on your team where everyone isn’t tightly interdependent.” It needs to be a foregone conclusion that “you’re not 100% available all the time.”
```

Comment:
@bobheadxi The calendar post you contributed to is almost ready to go live. Would you mind taking a quick glance at your section and letting me know if it's accurate?

Reply:
I left some comments: https://github.com/sourcegraph/about/pull/4607#pullrequestreview-761226565 Thank you so much! It's a bit odd reading about myself 😅